airflow/www/utils.py

```diff
@@ -246,7 +246,9 @@ def action_logging(f):
     """
     @functools.wraps(f)
     def wrapper(*args, **kwargs):
-        if current_user and hasattr(current_user, 'username'):
+        if current_user and hasattr(current_user, 'user'):
+            user = current_user.user.username
+        elif current_user and hasattr(current_user, 'username'):
```

Comment:
good catch! this seems like a bug and I wonder if we can remove this completely given it's always None.

Reply:
We use google authentication and `current_user.username` is always `None`, but I am not sure whether there is any other use cases or authentication methods that `current_user.username` is not `None`. I did a quick code search for `current_user.username` and it looks like there are some usage of `current_user.username` in UI. What do you think?

Reply:
Ah yeah, looks like the use case is trivial (airflow logo title) in the main UI, and it's currently returning `None` when you hover your mouse over it, so it can be replaced as well :)

Reply:
@jgao54 Done. I checked `current_user.username` for the rbac UI and it worked fine. I assume the bug only exits in the old UI.